### ISSUE TITLE
Not enough values to unpack when using compare in GUI

### ISSUE THREAD

**Issue by willhwillh** [danielhrisca/asammdf#720]
# Python version
python=3.9.13 (main, May 23 2022, 22:01:06) \n[GCC 9.4.0]'
'os=Linux-5.13.0-44-generic-x86_64-with-glibc2.31'
'numpy=1.22.3'
ldf is not supported
xls is not supported
xlsx is not supported
'asammdf=6.4.4'

# Code 

  ## MDF version
  4.11

  ## Code snippet

  1. In GUI: load two MDF files (and check that you can plot a channel for each)
  2. Under Mode > Comparison > Ctrl F, enter a valid channel and add to right hand pane for comparison. Select 'Plot' option.
  3. This then opens an error dialogue box
  
  ## Traceback
 
Error: <class 'ValueError'>	 
not enough values to unpack (expected 5, got 4)

Details: 2022-06-08, 11:46:35

<class 'ValueError'>	 
not enough values to unpack (expected 5, got 4)

  File "/home/will/.local/pipx/venvs/asammdf/lib/python3.8/site-packages/asammdf/gui/widgets/main.py", line 1172, in keyPressEvent
    self.add_window((ret, names))
  File "/home/will/.local/pipx/venvs/asammdf/lib/python3.8/site-packages/asammdf/gui/widgets/mdi_area.py", line 1167, in add_window
    flatten_entries = get_flatten_entries_from_mime(names)
  File "/home/will/.local/pipx/venvs/asammdf/lib/python3.8/site-packages/asammdf/gui/widgets/mdi_area.py", line 188, in get_flatten_entries_from_mime
    for (name, group_index, channel_index, mdf_uuid, type_) in data:

# Description

  1. In GUI: load two MDF files (and check that you can plot a channel for each)
  2. Under Mode > Comparison > Ctrl F, enter a valid channel and add to right hand pane for comparison. Select 'Plot' option.
  3. This then opens an error dialogue box, with stack trace shown above.

**Comment by danielhrisca** [comment]
Hello @willhwillh please try out the development branch code, or at least the latest released version 7.0.7

**Comment by willhwillh** [comment]
Thanks for the tip. Is there a preferred way of running the development branch documented anywhere?

The process I tried (after switching to the development branch) was:
```
python3.9 -m venv .venv
source .venv/bin/activate
pip install -r requirements_exe_build.txt
pip install -e .
```
I then tried to run ```asammdf/gui/asammdfgui.py``` which complained about missing modules PySide6, and pyqtlet2. After installing them the GUI window opens and I can plot a channel much as normal.

Aside: If I do the above on the master branch, the GUI opens as expected, but crashes when I try to plot a channel with:

```munmap_chunk(): invalid pointer```

I think this may be a known issue as I saw an issue which sounded similar iirc.
